quora-service/src/main/java/com/upgrad/quora/service/business/QuestionService.java

```diff
@@ -19,4 +24,29 @@ public QuestionEntity getQuestionByUuid(String questionUuid) throws InvalidQuest
         }
         return questionEntity;
     }
+
+    /** Authorize the delete operation on a question
+     * Only an owner or admin can delete the question
+     * @param questionEntity  question entity, containing owner details
+     * @param userEntity    user performing delete operation
+     * @param questionUuid    question's uuid, to be deleted
+     * @return  true if user can delete the answer
+     * @throws AuthorizationFailedException exception is thrown if user is not allowed to delete the answer
+     */
+    public boolean authorize(QuestionEntity questionEntity, UserEntity userEntity, String questionUuid) throws AuthorizationFailedException {
```

Comment:
remove questionUuid from the parameter as we are not using it .. I'll also remove same from AnswerService

Reply:
Made changes as per code review